operators/Makefile

```diff
@@ -12,7 +12,7 @@ ROOT_DIR = $(shell dirname $(CURDIR))
 export SHELL := /bin/bash
 
 KUBECTL_CLUSTER := $(shell kubectl config current-context 2> /dev/null)
-GKE_CLUSTER_VERSION ?= 1.11
+GKE_CLUSTER_VERSION ?= 1.12
```

Comment:
Does it mean we're not using GKE default version by default anymore?

Reply:
The GKE default is 1.12.8

Reply:
Oh nice 👍